src/vuex/modules/connection.js

```diff
@@ -20,26 +18,23 @@ export default function() {
   }
 
   const actions = {
-    async checkForPersistedNetwork({ dispatch, commit }) {
+    async checkForPersistedNetwork({ dispatch, commit }, apollo) {
```

Comment:
in line 4 you can use `export default function({ apollo }) {` as in `session.js`. It is provided to all modules.